homeassistant/components/mqtt/climate.py

```diff
@@ -216,6 +220,8 @@ def valid_preset_mode_configuration(config: ConfigType) -> ConfigType:
         vol.Optional(CONF_AUX_COMMAND_TOPIC): valid_publish_topic,
         vol.Optional(CONF_AUX_STATE_TEMPLATE): cv.template,
         vol.Optional(CONF_AUX_STATE_TOPIC): valid_subscribe_topic,
+        vol.Optional(CONF_CURRENT_HUMIDITY_TEMPLATE): cv.template,
+        vol.Optional(CONF_CURRENT_HUMIDITY_TOPIC): valid_subscribe_topic,
```

Comment:
Do we not need a `CONF_HUMIDITY_COMMAND_TOPIC` too? Further it would be good to add `CONF_MIN_HUMIDITY` and `CONF_MAX_HUMIDITY` as well.